test_scripts/API/SDL_Passenger_Mode/001_OnDriverDistraction_with_lockScreenDismissalEnabled_for_all_states.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0119-SDL-passenger-mode.md
+--
+-- Description:
+-- In case:
+-- 1) By policy OnDriverDistractions allowed for (FULL, LIMITED, BACKGROUND, NONE) HMILevel
+-- 2) App registered (HMI level NONE)
+-- 3) HMI sends "lockScreenDismissalEnabled" item (boolean) (with all mandatories) as a parameter of OnDriverDistraction notification
+-- 4) App activated (HMI level FULL)
+-- 5) HMI sends "lockScreenDismissalEnabled" item (boolean) (with all mandatories) as a parameter of OnDriverDistraction notification
+-- 6) Mobile app received OnHMIStatus (hmiLevel = "LIMITED")
+-- 7) HMI sends "lockScreenDismissalEnabled" item (boolean) (with all mandatories) as a parameter of OnDriverDistraction notification
+-- 8) Mobile app received OnHMIStatus (hmiLevel = "BACKGROUND")
+-- 9) HMI sends "lockScreenDismissalEnabled" item (boolean) (with all mandatories) as a parameter of OnDriverDistraction notification
```

Comment:
@HSavynetska Propose to leave one step for one level in order to have better understanding what script does.
And add a notice about other levels.